test/integration/default/inspec.yml

```diff
@@ -0,0 +1,6 @@
+name: fail2ban
```

Comment:
I've noticed this file in the repo: https://github.com/saltstack-formulas/fail2ban-formula/blob/master/Dockerfile.  Is there any way we could adapt that for `inspec` testing and then remove the file?  Or perhaps for another PR?

Reply:
I'll verify this and if easy to convert, I will. Else I'll let this for a future PR.

Reply:
done